core/controllers/admin_test.py

```diff
@@ -330,7 +330,7 @@ def test_regenerate_topic_related_opportunities_action(self):
         topic.subtopics = [
             topic_domain.Subtopic(
                 1, 'Title', ['skill_id_1'], 'image.svg',
-                constants.ALLOWED_THUMBNAIL_BG_COLORS['subtopic'][0],
+                constants.ALLOWED_THUMBNAIL_BG_COLORS['subtopic'][0], 0,
```

Comment:
Is 0 the right value here -- shouldn't we use a reasonable positive integer? One of the things I'm also wondering is whether we should also disallow a value of 0 when validating a PUT request.

Reply:
I agree, thumbnail_size_in_bytes should be None or "non-zero integer".